src/api/stats.ts

```diff
@@ -286,46 +283,15 @@ const getStatsForDetails = (
         .returns<CatalogStats_Details[]>();
 };
 
-export interface DefaultStatsWithDocument extends DefaultStats {
-    task_stats: object | null;
-}
-
-const getStatsForDashboard = (
-    tenant: string,
-    grain: Grains,
-    // endDate: DateTime,
-    duration?: Duration
-    // entityType?: Entity
-) => {
-    const endDate = DateTime.utc().startOf('month');
-
-    const past = duration ? endDate.minus(duration) : endDate;
-
-    // let query: string;
-    // switch (entityType) {
-    //     case 'capture':
-    //         query = CAPTURE_QUERY;
-    //         break;
-    //     case 'materialization':
-    //         query = MATERIALIZATION_QUERY;
-    //         break;
-    //     case 'collection':
-    //         query = COLLECTION_QUERY;
-    //         break;
-    //     default:
-    //         query = DEFAULT_QUERY;
-    // }
-
+const getStatsForDashboard = (tenant: string) => {
     return supabaseClient
         .from(TABLES.CATALOG_STATS)
-        .select(`${DEFAULT_QUERY},${TASK_STATS}`)
-        .like('catalog_name', `${tenant}%`)
-        .eq('grain', grain)
-        .or('bytes_written_by_me.gt.0,bytes_read_by_me.gt.0')
-        .gte('ts', past)
-        .lte('ts', endDate)
+        .select(`${DASHBOARD_QUERY}`)
+        .eq('catalog_name', `${tenant}`)
+        .eq('grain', 'monthly')
+        .eq('ts', DateTime.utc().startOf('month'))
```

Comment:
We can hard code a lot of these now. We ended up not adding the functionality to change the settings.

We'll always fetch the current month. There are some edge cases when a timezone is nearing a month change that will be weird but that should be okay.